### PR TITLE
Scripts/Commands: Cheats, Minor checks and changes

### PR DESCRIPTION
**Pull request by duhow**
Minor checks and changes of code.

Also more things before finishing...
How to detect status @ line 215 ?
Also, use ON / OFF in Explore Cheat @ line 251 ?

### CODE REVIEW

File: src/server/scripts/Commands/cs_cheat.cpp [hunk 1]
```diff
@@ -214,15 +210,17 @@ class cheat_commandscript : public CommandScript
...
         if (!*args)
         {
+            // Use Function player->isTaxiCheater() (bool)
+            // argstr = (handler->GetSession()->GetPlayer()->GetCommandStatus(CHEAT_TAXICHEAT)) ? "off" : "on";
             handler->SendSysMessage(LANG_USE_BOL);
             handler->SetSentErrorMessage(true);
             return false;
         }
 
-        std::string argstr = (char*)args;
```
Comment (jackpoz) [inline src/server/scripts/Commands/cs_cheat.cpp]: is this change required ?

Comment (Rochet2) [inline src/server/scripts/Commands/cs_cheat.cpp]: its bad to remove it since later it is compared: `argstr == "off"` and `char*` doesnt work like that.

**Comment by jackpoz** [comment]
broken travis build

**Comment by duhow** [comment]
It is WIP, please wait... (u.u)

**Comment by Vincent-Michael** [comment]
pls remove whitespace

File: src/server/scripts/Commands/cs_cheat.cpp [hunk 2]
```diff
@@ -214,29 +210,17 @@ class cheat_commandscript : public CommandScript
...
-
         Player* chr = handler->getSelectedPlayer();
 
         if (!chr)
             chr = handler->GetSession()->GetPlayer();
         else if (handler->HasLowerSecurity(chr, ObjectGuid::Empty)) // check online security
             return false;
-
-        if (argstr == "on")
+
```
Comment (Rochet2) [inline src/server/scripts/Commands/cs_cheat.cpp]: wspace

File: src/server/scripts/Commands/cs_cheat.cpp [hunk 3]
```diff
@@ -175,6 +175,8 @@ class cheat_commandscript : public CommandScript
         handler->PSendSysMessage(LANG_COMMAND_CHEAT_CT, player->GetCommandStatus(CHEAT_CASTTIME) ? enabled : disabled);
         handler->PSendSysMessage(LANG_COMMAND_CHEAT_POWER, player->GetCommandStatus(CHEAT_POWER) ? enabled : disabled);
         handler->PSendSysMessage(LANG_COMMAND_CHEAT_WW, player->GetCommandStatus(CHEAT_WATERWALK) ? enabled : disabled);
+        handler->PSendSysMessage(LANG_COMMAND_CHEAT_TAXINODES, player->isTaxiCheater() ? enabled : disabled);
+
```
Comment (tkrokli) [inline src/server/scripts/Commands/cs_cheat.cpp]: @duhow 	"Fix whitespace 3" (8 spaces). ;-)

Comment (duhow) [inline src/server/scripts/Commands/cs_cheat.cpp]: ................. Why don't YOU fix it? ¬¬ I'm editing in GitHub web, I can't see them...

Comment (jackpoz) [inline src/server/scripts/Commands/cs_cheat.cpp]: GitHub Web Editor is quite bad (i.e. it allows to create files with names starting with a space)

Comment (tkrokli) [inline src/server/scripts/Commands/cs_cheat.cpp]: The only way I could have changed anything is if you added me as "collaborator" in your repository, giving me write access to your files. It was meant as a friendly comment.
